packages/payments-client/src/components/pages/payment-session.tsx

```diff
@@ -0,0 +1,239 @@
+import React, { useEffect, useState } from 'react'
+import { Redirect, useParams, useLocation } from 'react-router'
+import useSWR from 'swr'
+import {
+  Section,
+  H3,
+  Helper,
+  Loader,
+  Formik,
+  Form,
+  CardInputGroup,
+  Input,
+  LevelRight,
+  Button,
+  unformatCard,
+  unformatCardExpires,
+  H5,
+  H6,
+  Grid,
+  GridItem,
+  notification,
+} from '@reapit/elements'
+import { PaymentModel } from '@reapit/foundations-ts-definitions'
+import { URLS } from '../../constants/api'
+import Routes from '../../constants/routes'
+import { MerchantKey, opayoSessionMerchantKeyService } from '../../opayo-api/merchant-key'
+import { opayoCreateTransactionServiceSession } from '../../opayo-api/transactions'
+import PaymentCustomerSection from '../ui/payments/payment-customer-section'
+// import PropertySection from '../ui/payments/property-section'
+import { localFetcher } from '../../utils/fetcher'
+import { updatePaymentStatus, UpdateStatusBody, UpdateStatusParams } from '../../services/payment'
+import { toastMessages } from '../../constants/toast-messages'
+
+export interface PaymentSessionModel extends PaymentModel {
+  clientCode: string
+}
+
+export const onUpdateStatus = async (body: UpdateStatusBody, params: UpdateStatusParams, result?: any) => {
+  const updateStatusRes = await updatePaymentStatus(body, params)
+
+  if (updateStatusRes) {
+    if (body.status === 'posted') {
+      return notification.success({
+        message: toastMessages.UPDATE_PAYMENT_STATUS_POSTED,
+        placement: 'bottomRight',
+      })
+    }
+    return notification.warn({
+      message: result?.description || toastMessages.UPDATE_PAYMENT_STATUS_REJECTED,
+      placement: 'bottomRight',
+    })
+  }
+
+  return notification.error({
+    message: toastMessages.FAILED_TO_UPDATE_PAYMENT_STATUS,
+    placement: 'bottomRight',
+  })
+}
+
+export const PaymentForm: React.FC<{
```

Comment:
Can you break the PaymentForm into it's own component so it can be shared between PaymentSessionPage and PaymentPage